### PR TITLE
:bug: (go/v4): fix make help instruction in the README

### PR DESCRIPTION
**Pull request by mateusoliveira43**
## Description
Update README template for golang/v4 plugin (v2 and v3 are deprecated and will be removed soon).

## Motivation
Use right command.

## Related issues
No related issues

### CODE REVIEW

**Review by camilamacedo86 — COMMENTED** [review]
Thank you for the contribution 🥇 

But you also need to run `make generate` so that we can update the samples under the testdata and used in the docs with this change.

Could you please run it and then ensure that you squash the commits for we get it merged?

/ok-to-test

File: pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 }
 
 const dockerfileTemplate = `# Build the manager binary
-FROM golang:1.20 as builder
+FROM golang:1.20 AS builder
```
Comment (camilamacedo86) [inline pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go:42]: Could we have a PR for each fix as well?
Could you please open another PR for this one?

Comment (mateusoliveira43) [reply pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go:42]: sure, will update this one and create the other one in a few

Comment (mateusoliveira43) [reply pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go]: done

**Comment by camilamacedo86** [comment]
Hi @mateusoliveira43 

For we are able to merge this you need to rebase it with the master branch and push. Can you please do that?

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

/approved

**Pull request merged**